Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_SimplePointer.cs

```diff
@@ -22,6 +22,8 @@ public class VRTK_SimplePointer : VRTK_WorldPointer
         public float pointerLength = 100f;
         public bool showPointerTip = true;
         public GameObject customPointerCursor;
+        [Tooltip("Check this if you would like to keep the custom pointer cursor material.")]
+        public bool keepCustomPointerMaterial = false;
```

Comment:
is it worth moving this into WorldPointer as it seems both SimplePointer and BezierPointer have it (and they both extend world pointer anyway)